component/src/main/java/org/exoplatform/rhmanagement/services/BalanceHistoryService.java

```diff
@@ -78,17 +78,17 @@ public void remove(BalanceHistoryDTO entity) {
     for (BalanceHistoryEntity entity : entities) {
      if(entity.getUpdaterId()!=null){
         try {
-          Profile profile = identityManager.getOrCreateIdentity(OrganizationIdentityProvider.NAME, entity.getUpdaterId(), false).getProfile();
+          Profile profile = identityManager.getOrCreateIdentity(OrganizationIdentityProvider.NAME, entity.getUpdaterId()).getProfile();
```

Comment:
profile declaration may be done in the begining of the for and it will be assigned to the returned object every time you will apply the identityManager.getOrCreateIdentity